app/views/static_pages/home.html.erb

```diff
@@ -1,12 +1,19 @@
-<h1>Langscope</h1>
-<p>Welcome to langscope!</p>
-<p>Please insert the languages that you speak<p>
-<%= form_for(:scope , url: myscope_path(:format => "json"), method: "get") do |f| %>
-   <div class="field">
-    <%= f.label :languages %><br>
-    <%= f.collection_select(:language_ids, Language.all, :id, :name,{}, {:multiple => true})  %>
-  </div>
-  <div class="actions">
-    <%= f.submit "Check my scope" %>
+<% if school_signed_in? %>
```

Comment:
It would be good to say `"Welcome, #{school.name}!"` or whatever attribute you're using.